src/libraries/System.Runtime.InteropServices/gen/DllImportGenerator/Microsoft.Interop.DllImportGenerator.props

```diff
@@ -21,7 +21,4 @@
       -->
     <CompilerVisibleProperty Include="DllImportGenerator_UseInternalUnsafeType" />
   </ItemGroup>
-  <PropertyGroup Condition="'$(SetDllImportGeneratorConstants)' != 'false'">
```

Comment:
We should be able to remove `SetDllImportGeneratorConstants` from `generators.targets`.